tests/test_identity.py

```diff
@@ -18,29 +17,70 @@
+# FAILED tests/test_identity.py::test_discrete[env2-PPO] - AssertionError: Mean reward below threshold: 5.90 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-RecurrentPPO] - AssertionError: Mean reward below threshold: 9.00 < 99.00
+
+# ** Apple M2: native, non-virtualized
+# FAILED tests/test_identity.py::test_discrete[env1-A2C] - AssertionError: Mean reward below threshold: 93.65 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-A2C] - AssertionError: Mean reward below threshold: 25.50 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-PPO] - AssertionError: Mean reward below threshold: 5.90 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-RecurrentPPO] - AssertionError: Mean reward below threshold: 8.90 < 99.00
+
+# * AMD EPYC: Flamingo
+# FAILED tests/test_identity.py::test_discrete[env1-A2C] - AssertionError: Mean reward below threshold: 93.10 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env1-PPO] - AssertionError: Mean reward below threshold: 71.05 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env1-RecurrentPPO] - AssertionError: Mean reward below threshold: 36.40 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-A2C] - AssertionError: Mean reward below threshold: 25.50 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-PPO] - AssertionError: Mean reward below threshold: 5.90 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-RecurrentPPO] - AssertionError: Mean reward below threshold: 7.15 < 99.00
+
+# * CircleCI (Intel?)
+# FAILED tests/test_identity.py::test_discrete[env1-A2C] - AssertionError: Mean reward below threshold: 93.10 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env1-PPO] - AssertionError: Mean reward below threshold: 71.05 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env1-RecurrentPPO] - AssertionError: Mean reward below threshold: 54.70 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-A2C] - AssertionError: Mean reward below threshold: 25.50 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-PPO] - AssertionError: Mean reward below threshold: 5.90 < 99.00
+# FAILED tests/test_identity.py::test_discrete[env2-RecurrentPPO] - AssertionError: Mean reward below threshold: 6.20 < 99.00
+
 
 @pytest.mark.parametrize("model_class", [A2C, PPO, DQN, RecurrentPPO])
 @pytest.mark.parametrize("env", [IdentityEnv(DIM), IdentityEnvMultiDiscrete(DIM), IdentityEnvMultiBinary(DIM)])
 def test_discrete(model_class, env):
-    torch.manual_seed(1234)
+    env_ = DummyVecEnv([lambda: env])
+    kwargs: dict[str, Any] = {}
+    n_steps = 10000
     if model_class == DQN:
-        TOTAL_TIMESTEPS = 10000
-        env_ = DummyVecEnv([lambda: copy.deepcopy(env)])
         kwargs = dict(learning_starts=0)
         # DQN only support discrete actions
         if isinstance(env, (IdentityEnvMultiDiscrete, IdentityEnvMultiBinary)):
             return
-    else:
-        TOTAL_TIMESTEPS = 50000
```

Comment:
This was an attempt to speed things up and make the test less flaky, so I've reverted it.